src/components/FindMovie/FindMovie.tsx

```
@@ -16,42 +63,59 @@ export const FindMovie: React.FC = () => {
               type="text"
               id="movie-title"
               placeholder="Enter a title to search"
-              className="input is-danger"
+              className={classNames('input', {
+                'is-danger': error,
+              })}
+              value={query}
+              onChange={event => {
+                setQuery(event.target.value);
+                setError(null);
```

The `onChange` handler clears `error` on input change which meets the requirement to hide the error when the title changes. Good.